main.py

```diff
@@ -36,21 +40,29 @@ def setup(self):
         self.link_up_timer = getattr(settings, 'LINK_UP_TIMER',
                                      DEFAULT_LINK_UP_TIMER)
 
-        self.verify_storehouse('switches')
-        self.verify_storehouse('interfaces')
-        self.verify_storehouse('links')
-
-        self.storehouse = StoreHouse(self.controller)
-
         self._lock = Lock()
         self._links_lock = Lock()
+        self.topo_controller = TopoController()
+        if self.ensure_db_or_core_shutdown():
+            self.topo_controller.bootstrap_indexes()
+            self.load_topology()
+
+    def ensure_db_or_core_shutdown(self) -> bool:
```

Comment:
This will be moved from here once https://github.com/kytos-ng/kytos/issues/198 is in place

Reply:
Since kytos core [PR 202 that solves issue 198](https://github.com/kytos-ng/kytos/pull/202) is under core review, I've already gone ahead and used it here, so no need for the `ensure_db_or_core_shutdown` on this NApp anymore.